src/components/FeatureNotification/OrganizationNotification.tsx

```diff
@@ -0,0 +1,93 @@
+import { Box } from '@mui/material';
+import { APP_PATHS } from 'src/constants';
+import strings from 'src/strings';
+import { Notification } from 'src/types/Notifications';
+import TextWithLink from 'src/components/common/TextWithLink';
+import { useOrganization, useTimeZones, useUser } from 'src/providers';
+import { getTodaysDateFormatted } from '@terraware/web-components/utils';
+import { useEffect, useState } from 'react';
+import isEnabled from 'src/features';
+import { InitializedTimeZone, TimeZoneDescription } from 'src/types/TimeZones';
+import { getTimeZone, getUTC } from 'src/utils/useTimeZoneUtils';
+import { OrganizationService, PreferencesService, UserService } from 'src/services';
+
+export default function OrganizationNotification(): Notification | undefined {
+  const { selectedOrganization, reloadOrganizations } = useOrganization();
+
+  const [timeZoneOrgNotification, setTimeZoneOrgNotification] = useState(false);
+  const [orgTimeZone, setOrgTimeZone] = useState<string>();
+
+  const timeZoneFeatureEnabled = isEnabled('Timezones');
+
+  const { user, reloadUser, userPreferences, reloadUserPreferences } = useUser();
+  const timeZones = useTimeZones();
+
+  useEffect(() => {
+    const getDefaultTimeZone = (): TimeZoneDescription => {
+      const browserTimeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+      return getTimeZone(timeZones, browserTimeZone) || getUTC(timeZones);
+    };
+
+    const notifyTimeZoneUpdates = (orgTz: InitializedTimeZone) => {
+      const notifyOrg = orgTz.timeZone && !orgTz.timeZoneAcknowledgedOnMs;
+      setOrgTimeZone(orgTz.timeZone);
+      setTimeZoneOrgNotification(!!notifyOrg);
+    };
+
+    const initializeTimeZones = async () => {
+      if (!user) {
+        return;
+      }
+
+      const userTz: InitializedTimeZone = await UserService.initializeTimeZone(user, getDefaultTimeZone().id);
```

Comment:
we don't want to run this multiple times... i can't find the discussion thread we had over this.

Reply:
One way to accomplish this... add a cached promise in UserService. Example:
```
/**
 * initialize user time zone
 */
const initializeTimeZone = async (user: User, timeZone: string): Promise<InitializedTimeZone> => {
  .. the usual code
};

/**
 * return a cached promise for initialized time zone
 * this avoids executing it twice
 * /
let initializedTimeZonePromise = null;
const getInitializedTimeZone = (): Promise<InitializedTimeZone> => {
  if (!initializedTimeZonePromise) {
    initializedTimeZonePromise = initializeTimeZone();
  }
  return initializedTimeZonePromise;
};
```
now the notifications code can call getInitiailizedTimeZone() (which returns a promise to the function that was only executed once).
just a suggestion.. if you have other ideas sure..